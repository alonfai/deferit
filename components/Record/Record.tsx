import React from 'react';
import { StyleSheet, Text, View } from 'react-native';

export type Props = {
  label: string;
  content: any;
};

const Recrod: React.FC<Props> = ({ label, content }) => {
  return (
    <View style={styles.container}>
      <Text style={styles.label}>{label}</Text>
      <Text style={styles.content}>{content}</Text>
    </View>
  );
};
const styles = StyleSheet.create({
  container: {
    flexDirection: 'row',
    marginVertical: 5,
  },
  label: {
    fontWeight: 'bold',
  },
  content: {
    paddingLeft: 5,
  },
});
export default Recrod;
