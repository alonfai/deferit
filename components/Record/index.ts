export { default } from './Record';
export type { Props } from './Record';
