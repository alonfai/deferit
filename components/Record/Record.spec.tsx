/// <reference types="@testing-library/jest-native" />
import React from 'react';
import { render } from '@testing-library/react-native';
import Record from './Record';

describe('<Header />', () => {
  test('Render successfully', async () => {
    const { toJSON } = render(<Record label='' content='' />);
    expect(toJSON()).toMatchSnapshot();
  });
});
