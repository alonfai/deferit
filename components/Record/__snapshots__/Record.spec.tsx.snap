// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> Render successfully 1`] = `
<View
  style={
    Object {
      "alignItems": "flex-start",
      "flexDirection": "row",
      "flexWrap": "wrap",
      "marginVertical": 5,
    }
  }
>
  <Text
    style={
      Object {
        "fontWeight": "bold",
      }
    }
  >
    
  </Text>
  <Text
    style={
      Object {
        "paddingLeft": 5,
      }
    }
  >
    
  </Text>
</View>
`;
